daemon-node.js

```diff
@@ -37,7 +37,7 @@ async function main () {
   }
 
   const cid = await client.dag.put(data, {
-    format: 'dag-test',
+    storeCodec: 'dag-test',
```

Comment:
I'm thinking that this should be changed to `client.block.put` to illustrate how to do this correctly with a custom codec, thoughts?

Reply:
The http client doesn't use `dag.get` either since the server has to serialize the output to transfer it over HTTP, instead it uses `block.get` internally while presenting the same API as core.

`dag.put` in the client should now probably do the same thing to maintain API compatibility.

This example might be a good place to show how to put/get dag objects using the block API as well as the DAG API though.